### DISCUSSION TITLE
Is it possible to create an executable for MAC of Ripes?

### DISCUSSION THREAD

**Discussion by Y1annis** [mortbopet/Ripes#283] [General]
Hello, I managed to create an executable using MSVC 2019 (windowsdeployqt) for windows but it doesnt work on MAC OS. Is there a way I can create an executable for MAC using my windows pc?

Thank you for your time

**Comment by mortbopet** [comment]
I don't personally build and run on osx, but we do that in CI. So you could theoretically just reproduce the steps in [this](https://github.com/mortbopet/Ripes/blob/master/.github/workflows/mac-release.yml) file, and/or enable the CI workflow in your Ripes branch in case you don't (like me) have access to a mac.

**Comment by Y1annis** [comment]
I am sorry for the late reply. I want to thank you for your answer!
